inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/resources/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Shouldn't this be at the root of the project?

Reply:
This is another file copied over from PDFAnno.
So I am not sure, I guess it should stay where it is as it applies to PDFAnno source only, and not the rest of this project.
I could also remove it, what do you think?

Reply:
It looks like some configuration file for the source code formatter in IntelliJ. @Rentier any idea? I think you added this kind of file somewhere as well.

Reply:
It is a cross IDE specification in how your project source code should be formatted. It should be unimportant.